package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "version": "0.1.0",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "tsdx test",
+    "lint": "tsdx lint",
+    "prepare": "tsdx build",
+    "size": "size-limit",
+    "analyze": "size-limit --why"
+  },
+  "peerDependencies": {},
+  "husky": {
+    "hooks": {
+      "pre-commit": "tsdx lint"
+    }
+  },
+  "prettier": {
+    "printWidth": 80,
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "es5"
+  },
+  "name": "sdk-v2",
+  "author": "Alexandru Matei",
+  "module": "dist/sdk-v2.esm.js",
+  "size-limit": [
+    {
+      "path": "dist/sdk-v2.cjs.production.min.js",
+      "limit": "10 KB"
+    },
+    {
+      "path": "dist/sdk-v2.esm.js",
+      "limit": "10 KB"
+    }
+  ],
+  "devDependencies": {
+    "@size-limit/preset-small-lib": "^7.0.8",
+    "husky": "^7.0.4",
+    "size-limit": "^7.0.8",
+    "tsdx": "^0.14.1",
```

Comment:
I dont think we want to do this differently to the other packages (we might want to adopt what you did here!). in other ts packages we use `tsc` without any helpers on it. why do we need `tsdx`? can we not just do it the same way we do it through out all the other Uma packages?

Reply:
Actually we are already using it for the `uma` SDK. It's an easy way to manage all the configurations/deps for setting up the compiler/IDE integration/linter

Reply:
one thing to mention about tsdx, its basically an opinionated way to setup a typescript library, but it does not play well with a monorepo, which is why it kind of sucks in protocol. But here as a standalone repo, i think its fine and does a lot of the boilerplate for you.